lib/features/sks-menu/presentation/widgets/sks_menu_loading.dart

```diff
@@ -0,0 +1,70 @@
+import "package:flutter/material.dart";
+import "package:lottie/lottie.dart";
+
+import "../../../../../theme/app_theme.dart";
+import "../../../../../utils/context_extensions.dart";
+import "../../../../gen/assets.gen.dart";
+
+class LottieLoading extends StatefulWidget {
+  const LottieLoading({
+    super.key,
+  });
+
+  @override
+  _LottieLoadingState createState() => _LottieLoadingState();
+}
+
+class _LottieLoadingState extends State<LottieLoading> {
+  final bool _showOldMenu = false;
+
+  void _gotoOldMenu() {
+    //Navigator.push(
+    //context,
+    //MaterialPageRoute(builder: (context) => const _SksMenuView()),
+    // );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Column(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          const Spacer(),
+          Center(
+            child: SizedBox(
+              width: 200,
+              height: 200,
```

Comment:
Some values can be separated to the ui_config as costant values.

Reply:
Actually I would reccomend using `SizedBox.square` constructor which removes duplication and config file extraction can prolly be skipped then.